src/NuGet.Clients/NuGet.PackageManagement.UI/ViewModels/OpenReadMeMarkdownViewModel.cs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NuGet.Common;
+
+namespace NuGet.PackageManagement.UI
+{
+    public class OpenReadMeMarkdownViewModel : INotifyPropertyChanged
+    {
+        private string _fileContent;
+        public OpenReadMeMarkdownViewModel(string filePath)
+        {
+            FileContent = "";
+        }
+
+        public string FileContent
+        {
+            get
+            {
+                return _fileContent;
+            }
+
+            private set
+            {
+                _fileContent = value;
```

Comment:
Check first to make sure value is different than _fileContent and only update _fileContent and push a PropertyChanged notification if it is.